src/extension_commands.ml

```diff
@@ -84,6 +84,30 @@ let switch_impl_intf =
   in
   command Extension_consts.Commands.switch_impl_intf handler
 
+module Dune_commands = struct
+  let open_current_dune_file =
+    let handler (_instance : Extension_instance.t) ~args:_ =
+      match Vscode.Window.activeTextEditor () with
+      | None ->
+        assert false
+        (* this command is available (in the command) palette only when a file
+           is open *)
+      | Some text_editor ->
+        let doc = TextEditor.document text_editor in
+        let uri = TextDocument.uri doc in
+        let dune_file_uri =
+          let path = Uri.fsPath uri |> Path.of_string in
+          let uri =
+            Path.relative path "../dune" |> Path.to_string |> Uri.file
+          in
+          Uri.toString uri ()
+        in
+        (open_file_in_text_editor dune_file_uri : TextEditor.t Promise.t)
+        |> ignore
```

Comment:
I'd prefer to use an explicit let binding here:
```ocaml
let (_ : TextEditor.t Promise.t) = ... in ()
```

Reply:
The current code seems terser (2 vs 4 lines) and slightly easier to comprehend (we don't do anything with the returned promise, so we ignore it explicitly) IMHO. Feel free to change it if you feel strongly about it.

Reply:
We use let bindings to ignore values in the rest of the codebase, so I like to stay consistent. I pushed the change.